test/testinput/bufr_ncep_rtma_prepbufr_adpupa.yaml

```diff
@@ -0,0 +1,227 @@
+          longName: "Pressure"
+          units: "Pa"
+
+        # ObsValue
+        - name: "ObsValue/airTemperature"
+          coordinates: "longitude latitude"
+          source: variables/airTemperature
+          longName: "Temperature"
+          units: "K"
+
+        - name: "ObsValue/dewPointTemperature"
+          coordinates: "longitude latitude"
+          source: variables/dewpointTemperature
+          longName: "Dewpoint Temperature"
+          units: "K"
+
+        - name: "ObsValue/virtualTemperature"
+          coordinates: "longitude latitude"
+          source: variables/virtualTemperature
+          longName: "Virtual Temperature Non-Q Controlled"
+          units: "K"
+
+        - name: "ObsValue/windEastward"
+          coordinates: "longitude latitude"
+          source: variables/windEastward
+          longName: "Eastward Wind"
+          units: "m s-1"
+
+        - name: "ObsValue/windNorthward"
+          coordinates: "longitude latitude"
+          source: variables/windNorthward
+          longName: "Northward Wind"
+          units: "m s-1"
+
+        - name: "ObsValue/specificHumidity"
+          coordinates: "longitude latitude"
+          source: variables/specificHumidity
+          longName: "Specific Humidity"
+          units: "kg kg-1"
+
+        # QualityMarker
+        - name: "QualityMarker/airTemperature"
+          coordinates: "longitude latitude"
+          source: variables/airTemperatureQualityMarker
+          longName: "Temperature Quality Marker"
+
+        - name: "QualityMarker/pressure"
+          coordinates: "longitude latitude"
+          source: variables/pressureQualityMarker
+          longName: "Pressure Quality Marker"
+
+        - name: "QualityMarker/windEastward"
+          coordinates: "longitude latitude"
+          source: variables/windQualityMarker
+          longName: "U, V-Component of Wind Quality Marker"
+
+        - name: "QualityMarker/specificHumidity"
+          coordinates: "longitude latitude"
+          source: variables/specificHumidityQualityMarker
+          longName: "Specific Humidity Quality Marker"
+
+        - name: "QualityMarker/height"
+          coordinates: "longitude latitude"
+          source: variables/heightQualityMarker
+          longName: "Height Quality Marker"
+
+        # ObsError
+        - name: "ObsError/airTemperature"
+          coordinates: "longitude latitude"
+          source: variables/airTemperatureObsError
+          longName: "Temperature Error"
+          units: "K"
+
+        - name: "ObsError/pressure"
+          coordinates: "longitude latitude"
+          source: variables/pressureObsError
+          longName: "Pressure Error"
+          units: "Pa"
+
+        - name: "ObsError/windEastward"
+          coordinates: "longitude latitude"
+          source: variables/windObsError
+          longName: "East and Northward wind error"
+          units: "m s-1"
+
+        - name: "ObsError/relativeHumidity"
```

Comment:
there is no `ObsValue` for `relativeHumidity`, there are 3 other moisture variables `dewpointTemperature`, `specificHumidity` and `virtualTemperature` having a error value for one or all of these would make sense

should figure out what is needed here but seem odd